presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -1202,6 +1208,7 @@ public PrestoS3OutputStream(
             this.sseType = requireNonNull(sseType, "sseType is null");
             this.sseKmsKeyId = sseKmsKeyId;
             this.requesterPaysEnabled = requesterPaysEnabled;
+            this.s3StorageClass = s3StorageClass;
```

Comment:
```suggestion
            this.s3StorageClass = requireNonNull(s3StorageClass, "s3StorageClass is null");
```

(here & other places)